cmd/res/profiles/profiles.yaml

```diff
@@ -0,0 +1,31 @@
+name: "Uart-Generic-Device"
```

Comment:
was this file renamed or copied?
Looks identical to https://github.com/edgexfoundry/device-uart/blob/main/cmd/res/profiles/device.uart.generic.yaml
file name should be `generic.yaml`. `profiles.yaml` name is ambiguous and and can only contain a single profile.

Reply:
Renamed res/profiles/device.uart.generic.yaml to res/profiles/generic.yaml